addons/resource.language.en_gb/resources/strings.po

```diff
@@ -7407,7 +7407,7 @@ msgstr ""
 
 #: xbmc/cores/AudioEngine/DSPAddons/ActiveAEDSP.cpp
 msgctxt "#15049"
-msgid "The audio DSP manager has been enabled without any enabled DSP add-on. Enable at least one add-on in order to use the audio DSP functionality."
+msgid "The DSP manager has been enabled without any enabled DSP add-on. Enable at least one add-on in order to use the DSP functionality."
```

Comment:
why i s "audio" dropped? There could be more DSP managers in the future and this gives no indication which